h/static/scripts/group-forms/components/AnnotationDocument.tsx

```diff
@@ -0,0 +1,37 @@
+import { AnnotationDocumentInfo } from '@hypothesis/annotation-ui';
+
+import { pageLabel } from '../utils/annotation-metadata';
+import type { APIAnnotationData } from '../utils/api';
+
+export type AnnotationDocumentProps = {
+  annotation: APIAnnotationData;
+};
+
+/**
+ * Render some metadata about an annotation's document and link to it
+ * if a link is available.
+ * Additionally, show the page number for annotations where it is available.
```

Comment:
```suggestion
 * if a link is available.
 *
 * Additionally, show the page number for annotations where it is available.
```

Add a blank line after the summary line.